client/components/comment.vue

```diff
@@ -58,7 +58,7 @@
                   flat
                   icon
                   small
-                  @click.stop="onDeleteClicked"
+                  @click="onDeleteClicked"
```

Comment:
Why this change? I like removal of `stop`, because stopping propagation is generally problematic (because often one depends on propagation somewhere else, like to close modals). But why is this change necessary here?

Reply:
I realized that if a comment is not focused and you click on those buttons it keeps defocused. A comment should be focused if you click on it (including the buttons).

Reply:
Yea. That is in general exactly the case why never use this. :-)